test/test_distributions.py

```diff
@@ -263,7 +297,7 @@ def test_uniform(self):
         self.assertEqual(uniform.log_prob(above_high).data[0], -float('inf'), allow_inf=True)
         self.assertEqual(uniform.log_prob(below_low).data[0], -float('inf'), allow_inf=True)
 
-        self._set_rng_seed(1)
+        set_rng_seed(1)
```

Comment:
Your comment at https://github.com/pytorch/pytorch/issues/4353#issuecomment-354020524 was extremely enlightening, and I think we should put it somewhere in the source code. Would you be willing to move this into a Note at the top of `test_distributions.py`, and then reference this note wherever we are explicitly calling `set_rng_seed`? Something like:

```
Note [Randomized statistical tests]
--------------
...
```

and then 

```
set_rng_seed(1)  # see Note [Randomized statistical tests]
```

Reply:
Done.